charts/eks-anywhere-packages/templates/awssecret.yaml

```diff
@@ -1,16 +1,14 @@
+{{- if not .Values.workloadOnly }}
```

Comment:
Will adding this change how helm updates work?

Reply:
I don't think so, I think that would depend on the annotation that Lewis was going to add. This doesn't really help or hurt that.

Reply:
Was this secret being used when created in namespaces other than the management one?

Reply:
After talking with Vincent, the previous version was never released so there shouldn't be any problem.